source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -230,7 +237,7 @@ namespace PlayFab
 
         // add batch to flight tracking map
         void* customData = reinterpret_cast<void*>(batchCounter); // used to track batches across asynchronous Events API
-        this->batchesInFlight[customData] = std::move(this->batch);
+        this->batchesInFlight[customData] = this->batch;
```

Comment:
>this->batch; [](start = 44, length = 12)

this now copies the vector `this->batch`

is that intentional? it doesn't seem valuable since the batch is cleared and on line 243

I think what you want to do is keep the std::move here and then change the clear line on 243 to just assert that the size is 0